src/app/components/FormElements/TimePicker.tsx

```diff
@@ -2,18 +2,26 @@ import React from 'react';
 import moment from 'moment';
 import Slider from 'rc-slider';
 import { TimeType } from '../../api/Api';
+import {
+  HandleWithTooltip,
+  createMarks,
+  sliderWithLimits
+} from './TimeSliderCommon';
 import 'rc-slider/assets/index.css';
-import { HandleWithTooltip, createMarks } from './TimeSliderCommon';
 
 export interface TimePickerProps {
   minTime: TimeType;
   maxTime: TimeType;
+  minAllowedTime?: TimeType;
```

Comment:
I forgot to take out the min/maxAllowedTime, these properties are now exposed by the sliderWithLimits enhancer